### ISSUE TITLE
Making more YouTube videos

### ISSUE THREAD

**Issue by jacobtomlinson** [dask/community#221]
As a side activity of #220 we've also been discussing how we can get more quality Dask videos on YouTube. It's a popular place for folks to go and learn and we can always add more.

The working group has some resources to have a videographer help us out with making some new content, whether this is editing editing content or adding new things entirely.

I'm opening this issue for two reasons. Firstly is to ask if folks have any video content that they would be interested in putting on the channel? This could be how-to videos, trainings, tutorials, anything that you may have made yourself to share information about Dask. We want the YouTube channel to be a real community resource and so want members of the community to feel comfortable submitting videos there.

Secondly it would be really good to think about how we could make a bite sized tutorial series on our channel. We currently have some recordings of tutorials on there (sadly in terrible Zoom recording quality) and bits and pieces that have been put together mostly by @mrocklin. I've been going some work with Kubeflow lately and really like how they have a [101 series on their youtube channel](https://www.youtube.com/playlist?app=desktop&list=PLIivdWyY5sqLS4lN75RPDEyBgTro_YX7x). Maybe we could do something similar for Dask?

We have another working group (myself, @jsignell, @ncclementi, @bryanwweber, @pavithraes, @ian-r-rose, @scharlottej13)  looking at refreshing the Dask tutorial for SciPy 2022 (#214). Perhaps we should create some video recordings of each chapter delivered separately and put it all together into a playlist of many ~10min videos on each of Dask's core topics. It would be awesome to get various members of the community presenting each chapter.

**Comment by scharlottej13** [comment]
This is a great idea! So far I'm most comfortable with the higher-level collections, but happy to help wherever it's needed.

**Comment by AdiReske** [comment]
@scharlottej13 and team, our Youtube channel looks a bit tired. If ok with everyone I'm going to put some time into it making it look better. Any suggestions? any new community videos you would like me to add?

**Comment by scharlottej13** [comment]
> any new community videos you would like me to add?

Thanks @AdiReske! Here are some we could add: 

- @GenevieveBuckley's [Scaling Science leveraging Dask for life sciences | SciPy 2021](https://www.youtube.com/watch?v=tY_lCGS1BMk) 
- @akbaritabar's [Parallelised analysis of large-scale bibliometric data with Dask in Python, DuckDB/DBeaver in SQL](https://www.youtube.com/watch?v=pYDVrBcIuYI) (Ali and I are former colleagues, and I've asked his permission)
- [Accelerated Cloud NLP for COVID-19 Research Using ScispaCy and Dask](https://www.youtube.com/watch?v=2EyNDWols-g) (I don't know the presenters, but seems like an interesting use of Dask for NLP)

**Comment by akbaritabar** [comment]
@scharlottej13 I had a look at my YouTube lists (I categorize things to find them again later) and noticed this video on dask and how to set up remote computing with docker images and then connecting to them through VS Code extensions. I am biased, but I found it helpful... in case.

[https://youtu.be/j6xYxUyMIKk](https://youtu.be/j6xYxUyMIKk)

**Comment by mrdanjlund77** [comment]
Hey everyone - My name is Danny and I'm the videographer @jacobtomlinson mentioned. I wanted to introduce myself and see how I can help with this project. I'm new to Dask, but great with video so I think we'll make a great team! Please let me know if you have any video ideas you'd like to talk about creating, or anything else content/video related that we can collaborate on together. Thanks!

**Comment by GenevieveBuckley** [comment]
Sure, I can send that to you @mrdanjlund77
The only contact details I could find for you is an email at the Ranch Studios, so I've shared it with that plus with Jacob and Sarah.

**Comment by mrdanjlund77** [comment]
Thanks @GenevieveBuckley ! I got your emails with the links:)

**Comment by GenevieveBuckley** [comment]
Hi @mrdanjlund77 I saw you uploaded some videos to the Dask channel. Can you update the descriptions to make it clear the content was first presented at SciPy 2021? Thanks :)

**Comment by mrdanjlund77** [comment]
Hey @GenevieveBuckley - all updated! Thanks for the heads up.

**Comment by GenevieveBuckley** [comment]
Who manages the Dask youtube channel? @mrdanjlund77 

As a quick heads up, I notice that [this youtube playlist](https://www.youtube.com/playlist?list=PLJ0vO2F_f6OAXBfb_SAF2EbJve9k1vkQX) now has a bunch of extra videos in it that don't belong. It's just the first six (so up to and including the Marine heatwaves tutorial) that were part of the dask down under workshop.

This might also suggest that other playlists could also be a bit mixed up - I'm not sure and haven't looked into that.

Context: noticed this while [fixing this broken youtube link](https://github.com/dask/dask-blog/pull/142)

**Comment by jsignell** [comment]
Thanks for mentioning that @GenevieveBuckley :) - @AdiReske can probably help sort the playlists out.

**Comment by mrdanjlund77** [comment]
@GenevieveBuckley I think the playlists are better organized now. I'll continue to keep an eye on them, but please let me know if you see more issues. I'll be managing it mostly now:)

Any suggestions or ideas for more interesting and helpful content we can make? Working on some tutorial ideas with @jacobtomlinson and @ncclementi and others...

**Comment by GenevieveBuckley** [comment]
> Any suggestions or ideas for more interesting and helpful content we can make? 

Basically, I think it'd be great to have stuff targeted towards intermediate users. Like ok, you've done the Dask introduction stuff and can write code, but how do you know if it's working well or not.

- I would find content on how to pick good settings for your use case (see https://github.com/dask/dask-blog/issues/118). 
- I half remember a converstaion where someone (maybe @jsignell) suggested that a mini tutorial or content on performance optimization would be good. Eg: take a Dask example, and have a look at what tweaks can be made to improve the performance.
- Matt's previous videos on how to use the dashboard to develop an intuition about performance, are pretty great. I think there's been a lot of extra graphs and new options added to the dashboard since then, so maybe more content about when and why you'd want to use those.
- Worked examples on debugging. It's hard to figure out why your Dask code might have crashed, because often the crash happens during an `execute` of a task, and it can be difficult to just drop into a debugger and inspect variables like you would usually. There is this page, which could be useful https://docs.dask.org/en/stable/how-to/debug.html and I think having some concrete examples of that would be very useful.

**Comment by mrdanjlund77** [comment]
This is great! Thanks @GenevieveBuckley

**Comment by mrdanjlund77** [comment]
Yeah thanks Genevieve. I’ll take a look. I hadn’t messed with that list - it was before my time. But I can take a look. 

Any suggestions? 

Also wondering if you tracked down the other broken link issue?
